tests/vm_test.go

```diff
@@ -1921,6 +1923,72 @@ status:
 			}, 2*time.Minute, 1*time.Second).Should(Succeed())
 		})
 	})
+
+	Context("[Serial] when node becomes unhealthy", Serial, func() {
+		const componentName = "virt-handler"
+		var nodeName string
+		var numberOfVirtHandler int32
+
+		handlerPodGetter := func(virtCli kubecli.KubevirtClient, componentName, namespace, nodeName string) *k8sv1.PodList {
```

Comment:
Again, this adds an unnecessary closure.

Reply:
Same as above.